ethereum/PFP.sol

```diff
@@ -147,14 +147,15 @@ contract PFP is ERC721A, Ownable, VRFConsumerBase {
             if (MerkleProof.verify(merkleProof, snapshotMerkleRoot, keccak256(abi.encodePacked(msg.sender, '_', allowance)))) {
                 if(_claimedAmount[coldWallet] + numberOfTokens <= allowance ) {
                     _safeMint(msg.sender, numberOfTokens);
-                    result = string(abi.encodePacked("Mint is successed!"));
+                    _claimedAmount[coldWallet] += numberOfTokens;
+                    result = string(abi.encodePacked("Mint successful!"));
                     return result;
                 } else {
                     isVerified = true;
                 }
             }
         }
-        result = isVerified? string(abi.encodePacked("Claim would exceed allowance of tokens!")) : string(abi.encodePacked("Address is not verified in snapshot!"));
+        result = isVerified? string(abi.encodePacked("Claim exceeds token allowance!")) : string(abi.encodePacked("Address not in snapshot!"));
```

Comment:
Very minor - can save a bit of gas on the calldata by getting these strings under 32 bytes each.